bin/tflocal

```diff
@@ -241,6 +269,25 @@ def generate_s3_backend_config() -> str:
     return result
 
 
+def check_override_file(providers_file):
+    """Checks override file existance"""
+    try:
+        if os.path.exists(providers_file):
+            msg = f"Providers override file {providers_file} already exists"
+            err_msg = msg + " - please delete it first"
+            if DRY_RUN:
+                msg += ". File will be overwritten."
+                print(msg)
+                print("\tOnly 'yes' will be accepted to approve.")
+                if input("\tEnter a value: ") != "yes":
+                    raise FileExistsError(err_msg)
+            else:
+                raise FileExistsError(err_msg)
+    except FileExistsError as e:
+        print(e)
```

Comment:
Would generally try to contextualize these error logs a bit and provide a human readable message, rather than printing the raw exception string.

Maybe something like:
```
print(f"Provider override file {providers_file} already exists, exiting ...")
```

Also, do you think we need the additional try/except blocks here? We could probably simply `print` & `exit` separately in the two relevant branches above 👍

Reply:
The error message itself was human-readable but you are right, it can be simplified.